src/Microsoft.DocAsCode.Build.SchemaDriven/ApplyOverwriteFragments.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.DocAsCode.Build.SchemaDriven
+{
+    using System.Collections.Generic;
+    using System.Collections.Immutable;
+    using System.Composition;
+    using System.Linq;
+
+    using Microsoft.DocAsCode.Build.Common;
+    using Microsoft.DocAsCode.Common;
+    using Microsoft.DocAsCode.Plugins;
+
+    [Export(nameof(SchemaDrivenDocumentProcessor), typeof(IDocumentBuildStep))]
+    public class ApplyOverwriteFragments : BaseDocumentBuildStep, ISupportIncrementalBuildStep
```

Comment:
>ApplyOverwriteFragments [](start = 17, length = 23)

add ut for overwrite & incremental overwrite

Reply:
UT will be added after Renze's next PR

---
In reply to: [166499334](https://github.com/dotnet/docfx/pull/2419#discussion_r166499334) [](ancestors = 166499334)